### ISSUE TITLE
ESM version for modern browsers

### ISSUE THREAD

**Issue by jehna** [jehna/ga-lite#253]
![image](https://user-images.githubusercontent.com/696957/111087888-7a579400-84e1-11eb-97b8-1c2cc77e259d.png)

_Originally posted by @datapolitical in https://github.com/jehna/ga-lite/issues/252#issuecomment-798999591_

More info: https://philipwalton.com/articles/deploying-es2015-code-in-production-today/

**Comment by datapolitical** [comment]
I would love to. But I wouldn’t have the first clue where to start.

**Comment by jehna** [comment]
In theory you could also point the `<script type="module">` to the `src/ga-lite.js` and it should work just fine. The only issue is, that it does not expose a global ga-lite instance (like IMO the ESM should work in the first place). So you would have to have something like:

```
<script type="module">
import galite from 'path/to/galite/sec/ga-lite.js';

galite('create', 'UA-XXXXXXXX-X', 'auto');
galite('send', 'pageview');
</script>
```

It does get much more complicated if you are tracking more than just page views, but in that case you probably want to separate ESM/es5 versions from your own code too

**Comment by Gonzo2O28** [comment]
Any news or suggestions in how to fix this?

**Comment by jehna** [comment]
@carlmjohnson Thank you for noticing! Should be fixed now with 08a96bb and fixed version 2.1.4 should be available from npm now

**Comment by carlmjohnson** [comment]
Fast, thanks.
